Add interface esi settings (#134)
jeremmfr/terraform-provider-junos: website/docs/r/interface_physical.html.markdown

* `ae_link_speed` - (Optional)(`String`) Link speed of individual interface that joins the AE.
* `ae_minimum_links` - (Optional)(`Int`) Minimum number of aggregated links (1..8).
* `description` - (Optional)(`String`) Description for interface.

> * `esi` -(Optional)(`ListOfString`)([attribute-as-blocks mode](https://www.terraform.io/docs/configuration/attr-as-blocks.html)) Define ESI Config parameters. See the [`esi` arguments](#esi-arguments) block.

Reviewer: ```suggestion
* `esi` -(Optional)([attribute-as-blocks mode](https://www.terraform.io/docs/configuration/attr-as-blocks.html))  Can be specified only once to define ESI Config parameters. See the [`esi` arguments](#esi-arguments) block.
```

* `ether802_3ad` - (Optional)(`String`) Name of aggregated device for add this interface to link of 802.3ad interface.
* `trunk` - (Optional)(`Bool`) Interface mode is trunk.
* `vlan_members` - (Optional)(`ListOfString`) List of vlan for membership for this interface.
* `vlan_native` - (Optional)(`Int`) Vlan for untagged frames.
* `vlan_tagging` - (Optional)(`Bool`) Add 802.1q VLAN tagging support.

---

> #### esi-arguments

Reviewer: Missing `mode`argument

> * `identifier` - (Required)(`String`) The ESI value for the interface

Reviewer: Is Optional

Author: Is required.. Fixed Schema.

> * `auto-derive` - (Optional)(`String`) Auto-derive ESI value for the interface

Reviewer: Need to be `auto_derive_lacp` and type `bool`

> * `ef-election-type` - (Optional)(`String`) DF Election Type

Reviewer: Typo, need to be `df_election_type`

* `ae_link_speed` - (Optional)(`String`) Link speed of individual interface that joins the AE.
* `ae_minimum_links` - (Optional)(`Int`) Minimum number of aggregated links (1..8).
* `description` - (Optional)(`String`) Description for interface.

> * `esi` -(Optional)(`ListOfString`)([attribute-as-blocks mode](https://www.terraform.io/docs/configuration/attr-as-blocks.html)) Can be specified only once to define ESI Config parameters. See the [`esi` arguments](#esi-arguments) block.

Reviewer: (`ListOfString`) must be remove
```suggestion
* `esi` -(Optional)([attribute-as-blocks mode](https://www.terraform.io/docs/configuration/attr-as-blocks.html)) Can be specified only once to define ESI Config parameters. See the [`esi` arguments](#esi-arguments) block.
```

* `ether802_3ad` - (Optional)(`String`) Name of aggregated device for add this interface to link of 802.3ad interface.
* `trunk` - (Optional)(`Bool`) Interface mode is trunk.
* `vlan_members` - (Optional)(`ListOfString`) List of vlan for membership for this interface.
* `vlan_native` - (Optional)(`Int`) Vlan for untagged frames.
* `vlan_tagging` - (Optional)(`Bool`) Add 802.1q VLAN tagging support.

---

> #### esi-arguments

Reviewer: It should be `esi arguments`
```suggestion
#### esi arguments
```

> * `identifier` - (Optional)(`String`) The ESI value for the interface
> * `auto_derive_lacp` - (Optional)(`Bool`) Auto-derive ESI value for the interface
> * `df_election_type` - (Optional)(`String`) DF Election Type
> * `mode` - (Required)(`String`) ESI Mode
> * `source_bmac` - (Optional)(`String`) Unicast Source B-MAC address per ESI for PBB-EVPN

Reviewer: `mode` shoud be in first because is required
```suggestion
* `mode` - (Required)(`String`) ESI Mode
* `identifier` - (Optional)(`String`) The ESI value for the interface
* `auto_derive_lacp` - (Optional)(`Bool`) Auto-derive ESI value for the interface
* `df_election_type` - (Optional)(`String`) DF Election Type
* `source_bmac` - (Optional)(`String`) Unicast Source B-MAC address per ESI for PBB-EVPN
```
